src/label/Label.ts

```diff
@@ -30,17 +30,30 @@ const labelDefaults = {
  *
  * Properties that can be set on a Label component
  *
- * @property classes    Optional classes to be set on the label node
  * @property formId     ID of a form element associated with the form field
  * @property label      Label settings for form label text, position, and visibility
  */
 export interface LabelProperties extends ThemeableProperties {
 	registry?: WidgetRegistry;
-	classes?: ClassesFunctionChain;
 	formId?: string;
 	label: string | LabelOptions;
 }
 
+/**
+ * This is a helper function for using `extraClasses` with Label.
+ * It can be used as follows:
+ * extraClasses: { root: parseLabelClasses(this.classes(css.class1, css.class2).get()) }
+ */
+export function parseLabelClasses(classes: { [key: string]: boolean }): string {
+	return Object.keys(classes).reduce((classNamesString: string, className) => {
+		if (classes[className] === true) {
+			classNamesString += ` ${className}`;
+		}
+
+		return classNamesString;
+	}, '').trim();
+}
+
 export const LabelBase = RegistryMixin(ThemeableMixin(WidgetBase));
 
 @theme(baseCss)
```

Comment:
Finally, to enable this to actually work and to finish of #75 Could you please create `label.m.css` under the `label/styles` directory, change this to `@theme(css)` and pass an empty `css.root` to the label's root node please. Otherwise `extraClasses` will have no target `root` to apply the classes to.